src/main/java/com/marianbastiurea/lifeofbees/Environment.java

```diff
@@ -7,12 +7,15 @@
 
  */
 
+import com.marianbastiurea.lifeofbees.weather.Weather;
+
 public class Environment {
     private Weather weather;
 
     private Square[][] terrain = new Square[100][100];
```

Comment:
Simplify structure by removing terrain and `Square` class. Move 
```
private Queen queen;
 private List<Bee> bees;
 private Hive hive;
```
fields  from `Square`class  to `LifeOfBees` class

Reply:
delete row 15, yes?

Reply:
done